packages/ols/ols.yaml

```
@@ -0,0 +1,33 @@
+---
+name: ols
+description: Language server for Odin. This project is still in early development.
+homepage: https://github.com/DanielGavin/ols
+licenses:
+  - MIT
+languages:
+  - Odin
+categories:
+  - LSP
+
+source:
+  id: pkg:github/DanielGavin/ols@nightly
+  asset:
+    - target: [darwin_x64, darwin_arm64]
+      file: ols-x86_64-darwin
+      bin: ols-x86_64-darwin
+    - target: linux_x64
+      file: ols-x86_64-unknown-linux-gnu 
+      bin: ols-x86_64-unknown-linux-gnu
+    - target: linux_x86
+      file: ols-x86_64-unknown-linux-gnu 
+      bin: ols-x86_64-unknown-linux-gnu
```

Or wait, these are GNU binaries for x64 only

```suggestion
    - target: linux_x64_gnu
      file: ols-x86_64-unknown-linux-gnu 
      bin: ols-x86_64-unknown-linux-gnu
```